source/write/bulk-write.txt

```diff
@@ -15,43 +15,346 @@ Bulk Write Operations
+By default, these methods run the operations in the order you defined
+when constructing ``ClientBulkWrite``.
+
+The following code demonstrates how to use the ``bulkWrite()`` method
+to perform a bulk write operation on multiple namespaces:
+
+.. io-code-block::
+   :copyable:
+
+   .. input:: /includes/write/bulk-write.php
+      :start-after: start-bulk-client
+      :end-before: end-bulk-client
+      :language: php
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      Inserted documents: 2
+      Modified documents: 2
+      Deleted documents: 200
+
+.. _php-client-bulk-write-options:
+
+Customize Bulk Write
+~~~~~~~~~~~~~~~~~~~~
+
+You can modify the behavior of the client bulk write operation by
+passing an array to the ``ClientBulkWrite`` constructor that specifies
+option values as a parameter. The following table describes the options
+you can set in the array:
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Option
+     - Description
+
+   * - ``bypassDocumentValidation``
+     - | Specifies whether the operation bypasses document validation. This lets you 
+         modify documents that don't meet the schema validation requirements, if any 
+         exist. For more information about schema validation, see :manual:`Schema
+         Validation </core/schema-validation/#schema-validation>` in the {+mdb-server+}
+         manual.
+       | Defaults to ``false``.
+
+   * - ``comment``
+     - | Attaches a comment to the operation. For more information, see the :manual:`insert command
+         fields </reference/command/insert/#command-fields>` guide in the
+         {+mdb-server+} manual.
+
+   * - ``let``
+     - | Specifies a document with a list of values to improve operation readability.
+         Values must be constant or closed expressions that don't reference document
+         fields. For more information, see the :manual:`let statement
+         </reference/command/update/#std-label-update-let-syntax>` in the
+         {+mdb-server+} manual.
+
+   * - ``ordered``
+     - | If set to ``true``: when a single write fails, the operation stops without
+         performing the remaining writes and throws an exception.
+       | If set to ``false``: when a single write fails, the operation continues to
+         attempt the remaining write operations, if any, then throws an exception.
+       | Defaults to ``true``.
+
+   * - ``verboseResults``
+     - | Specifies whether to return verbose results. 
+       | Defaults to ``false``.
+
+The following example creates a ``ClientBulkWrite`` instance and sets
+the ``ordered`` option to ``false``:
+
+.. literalinclude:: /includes/write/bulk-write.php
+   :start-after: start-bulk-client-options
+   :end-before: end-bulk-client-options
+   :language: php
+   :dedent:
+
+If the library runs the insert operation first, one document is deleted.
+If it runs the delete operation first, no documents are deleted. 
```

Comment:
Q: which insert and delete operation are you referring to here? do you just mean any insert and delete where the delete operation targets that inserted document?

Reply:
This might be a copypaste issue - I will remove

Reply:
I changed the note in this section to describe the same situation